utils/spacewalk-common-channels.ini

```diff
@@ -3675,6 +3675,253 @@ checksum = sha256
 base_channels = debian-12-pool-amd64-uyuni
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian12-Uyuni-Client-Tools/Debian_12/
 
+[raspbianos-12-pool-arm64-uyuni]
```

Comment:
Unfortunately there's no RaspbianOS, name is Raspberry Pi OS: https://www.raspberrypi.com/software/operating-systems/ so we should use the official name here and at the other places (except when detecting the codename, as armhf still uses `raspbian` and arm64 uses `debian`).

Reply:
Changed everywhere (which were a lot of entries, I trusted search and replace + a manual inspection later on). Hope nothing's missing.